[33m@practica/error-handling:prepare[0m: cache hit, replaying output [2m12fc880bee56fe0d[0m
[33m@practica/error-handling:prepare: [0m
[33m@practica/error-handling:prepare: [0m> @practica/error-handling@0.0.2 prepare
[33m@practica/error-handling:prepare: [0m> npm i && npm run build
[33m@practica/error-handling:prepare: [0m
[33m@practica/error-handling:prepare: [0mnpm ERR! process terminated
[33m@practica/error-handling:prepare: [0mnpm ERR! signal SIGINT
[33m@practica/error-handling:prepare: [0m
[33m@practica/error-handling:prepare: [0mnpm ERR! A complete log of this run can be found in:
[33m@practica/error-handling:prepare: [0mnpm ERR!     /Users/yonigoldberg/.npm/_logs/2022-07-20T07_47_22_515Z-debug-0.log
